d1lod/scripts/benchmark.py

```diff
@@ -2,8 +2,8 @@
 
 import uuid
 
-s = sesamestore.SesameStore("localhost", 8080)
-r = sesamestore.SesameRepository(s, "test")
+s = sesamestore.SesameStore("virtuoso", 8890)
```

Comment:
This seems like it'd no longer be relevant since we stripped out Sesame. I'll take a look at this after merge.